src/plugins/d3/renderer/hooks/useSeries/utils.ts

```diff
@@ -18,16 +19,14 @@ export const getAllLegendItems = (series: PreparedSeries[]) => {
     return series.map((s) => s.name);
 };
 
-export function prepareLegendSymbol(series: ChartKitWidgetSeries): PreparedLegendSymbol {
-    const symbolOptions: RectLegendSymbolOptions = series.legend?.symbol || {};
-    const symbolHeight = symbolOptions?.height || DEFAULT_LEGEND_SYMBOL_SIZE;
+export function prepareLegendSymbol(series: ChartKitWidgetSeries, index = 0): PreparedLegendSymbol {
+    const scatterStyle = (series as ScatterSeries).symbol || getScatterStyle(index);
 
     return {
-        shape: 'rect',
-        width: symbolOptions?.width || DEFAULT_LEGEND_SYMBOL_SIZE,
-        height: symbolHeight,
-        radius: symbolOptions?.radius || symbolHeight / 2,
-        padding: symbolOptions?.padding || DEFAULT_LEGEND_SYMBOL_PADDING,
+        shape: 'symbol',
+        style: scatterStyle,
+        padding: 0,
+        width: 8,
```

Comment:
Why are these values(padding and width) not taken from the settings but are hard-coded?